### PR TITLE
add story for header

### LINKED ISSUE #2196: Add container story for HeaderContainer
**Is your feature request related to a problem? Please describe.**
As we have moved components over to the Psammead component library, we have few items in our Simorgh Storybook: https://bbc.github.io/simorgh/

We should include our container components in the Simorgh storybook.

**Describe the solution you'd like**
Update story to Header Container (on storybook)

- [ ] What would be ideal, would be to add a drop-down that could switch between services and populate these with the correct view. e.g. selecting Persian from the drop-down when looking at the BrandContainer will show you the News | Persian brand.
- [ ] The containers relating to RTL services, (e.g. Persian and in future, other RTL services) should have the `dir="rtl"` attribute passed through to Storybook. Could be added via a Decorator, that gets the value of `dir` from the Service Context. 

See the implementation of this for the Psammead sitewide links component - https://bbc.github.io/psammead/?path=/story/components-sitewidelinks--default When you select `Persian` from the `language` dropdown, it loads links in the right language and adds the correct `dir` value too.

**Describe alternatives you've considered**

**Testing notes**
Regression of the HeaderContainer

Dev insight: Cypress tests for Storybook may need to be updated, if so pair with test.

**Additional context**
We shouldn't need stories for all of our containers (e.g. Blocks, or Fragments) since these are for handling our Optimo data.

### PR DESCRIPTION
**Pull request by thekp**
Resolves #2196

**Overall change:** _Update header container storybook to have a dropdown of services and have RTL direction for required services._

**Code changes:**

- added storybook-helper package
- added knobs for the footer storybook

note: there is no config for navigation for persian/news, hence the navigation does not show in the header story for it

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/Header/index.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  .add(
+    'default',
+    inputProvider(
+      null,
+      ({ service }) => {
+        const serviceContextStub = {
+          product: 'BBC News',
+          service,
+          brandSVG: svgs[service],
+          svgHeight: 24,
```
Comment (dr3) [inline src/app/containers/Header/index.stories.jsx]: I dont think these 3 values need to be in the service context stub, they arnt in the real service configs, the values are inside the brand container

**Review by dr3 — CHANGES_REQUESTED** [review]
Just 1 small thing :)

**Review by ryanmccombe — APPROVED** [review]

File: src/app/containers/Header/index.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    inputProvider(
+      null,
+      ({ service }) => {
+        const serviceContextStub = {
+          product: 'BBC News',
+          service,
+          brandSVG: svgs[service],
+        };
+        return (
+          <ServiceContextProvider service={service} value={serviceContextStub}>
```
Comment (dr3) [inline src/app/containers/Header/index.stories.jsx]: ServiceContextProvider doesnt take value, do you mean ServiceContext.Provider? if so, it doesnt take service

Comment (thekp) [reply src/app/containers/Header/index.stories.jsx]: i see,  updating now to use ServiceContext.Provider as the container does not require service prop anyways

**Review by dr3 — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
This doesn't need testing, it's just adding a story

**Pull request merged**